desktop/src/app/components/navbar/warnings/modals/fix-outliers-modal.html

```diff
@@ -9,12 +9,24 @@ <h5 class="modal-title" i18n="@@navbar.taskbar.fixOutliers.header">
         Bitte wählen Sie den korrekten Wert aus der Werteliste aus.
     </div>
 
-    <searchable-select [selectedValue]="selectedValue"
-                       [values]="getValues()"
-                       [getLabel]="getValueLabel"
-                       (onValueSelected)="selectedValue = $event"></searchable-select>
+    <div *ngIf="inputType != 'checkboxes'">
+        <searchable-select [selectedValue]="selectedValues[0]"
+                           [values]="getValues()"
+                           [getLabel]="getValueLabel"
+                           (onValueSelected)="selectedValues.push($event)">
```

Comment:
The list should be completely replaced with a new list that only contains the selected value (or the old value deleted first). Otherwise this leads to the bug that always the value you selected first in the dropdown list is used for the replacement. If you select a different value from the list afterwards, this change is not considered.